matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/registration/DefaultRegistrationWizard.kt

```diff
@@ -45,6 +47,7 @@ internal class DefaultRegistrationWizard(
     private val registerAvailableTask: RegisterAvailableTask = DefaultRegisterAvailableTask(authAPI)
     private val registerAddThreePidTask: RegisterAddThreePidTask = DefaultRegisterAddThreePidTask(authAPI)
     private val validateCodeTask: ValidateCodeTask = DefaultValidateCodeTask(authAPI)
+    private val registerOtherTask: RegisterOtherTask = DefaultRegisterOtherTask(authAPI)
```

Comment:
Can you rename `Other` to `Custom` please (so 3 times here)?

Reply:
done